### PR TITLE
Makes cheese effect temporary, also change back on cheese mob death

### PR DESCRIPTION
**Pull request by JamieD1**
# Github documenting your Pull Request

Makes cheese effect temporary, also change back on cheese mob death

# Wiki Documentation

Makes cheese effect temporary, also change back on cheese mob death

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
tweak: Makes cheese effect temporary, also change back on cheese mob death
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
this is the part where cheese staff gets added normally.............................

**Comment by JamieD1** [comment]
> this is the part where cheese staff gets added normally.............................

Soon

File: code/modules/mob/living/simple_animal/friendly/cheese.dm [hunk 1]
```diff
@@ -21,6 +21,7 @@
 	speed = 5
 	can_be_held = TRUE
 	density = FALSE
+	var/mob/living/carbon/human/stored_mob
```
Comment (adamsong) [inline code/modules/mob/living/simple_animal/friendly/cheese.dm]: cheesify takes a `mob/living`

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/friendly/cheese.dm (hunk 1 above)
Comment (JamieD1) [reply code/modules/mob/living/simple_animal/friendly/cheese.dm]:
```suggestion
	var/mob/living/stored_mob
```

**Review by adamsong — APPROVED** [review]

**Pull request merged**